keymanapp/keyman
# chore(common): Update Crowdin strings for Fula

Updates Crowdin strings for Fula (ff).

TODO
- [x] Locale is new for macOS, so @sgschantz will need to run it through XCode

## User Testing
Wait for @sgschantz to complete the above TODO before attempting to test.

* **TEST_MACOS**
1. Start Keyman for macOS
2. Set the system locale to Fula (ff)
3.  Verify Keyman UI is in Fula
